changelog.d/pulsar_sink_source_support_tls_options.feature.md

```diff
@@ -0,0 +1,3 @@
+Tls options to set custom certificate chain are now available for `pulsar` sink and source.
```

Comment:
```suggestion
The `pulsar` source and sink now support configuration of TLS options via the `tls` configuration field.
```

Cleaning this up a bit.

Reply:
fixed in https://github.com/vectordotdev/vector/pull/22148/commits/85de10c5985602d1a4cf8408b7adaf355a28db8b